web/crux-ui/Dockerfile

```diff
@@ -8,30 +8,31 @@ COPY ./package.json ./package-lock.json ./.npm ./
 RUN npm ci --arch=x64 --platform=linuxmusl --cache .npm --prefer-offline --no-fund
 
 COPY . .
-RUN npm run build && \
-    npm prune --production
+RUN npm run build
 
 # stage 2, run the app
-FROM docker.io/library/node:19.2-alpine AS RUNNER
+FROM docker.io/library/node:20-alpine3.17 AS RUNNER
 WORKDIR /app
 
 ENV NODE_ENV production
 ENV NEXT_TELEMETRY_DISABLED 1
 ENV TZ UTC
-RUN apk --update add tzdata
+RUN apk --update add tzdata && \
+    mkdir .next && \
+    chown node:node .next
 
+COPY --from=BUILDER --chown=node:node /app/.next/standalone ./
 COPY --from=BUILDER --chown=node:node /app/public ./public
-COPY --from=BUILDER /app/package.json ./package.json
-COPY --from=BUILDER --chown=node:node /app/.next ./.next
-COPY --from=BUILDER /app/node_modules ./node_modules
+COPY --from=BUILDER --chown=node:node /app/.next/static ./.next/static
 
 USER node
 
 EXPOSE 3000
+ENV HOSTNAME "0.0.0.0"
 ENV PORT 3000
 LABEL org.opencontainers.image.source="https://github.com/dyrector-io/dyrectorio/web/crux-ui"
 
 HEALTHCHECK --interval=30s --timeout=5s \
     CMD wget -nv -t1 --spider http://localhost:3000/ || exit 1
 
-ENTRYPOINT [ "npm", "run", "start:prod"]
+CMD [ "node", "server.js" ]
```

Comment:
Using the ENTRYPOINT directive is preferable over CMD, though technically there is no much difference how it is executed.

Reply:
We had a discussion that we need the entrypoint from the original image.
```sh
#!/bin/sh
set -e

# Run command with node if the first argument contains a "-" or is not a system command. The last
# part inside the "{}" is a workaround for the following bug in ash/dash:
# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=874264
if [ "${1#-}" != "${1}" ] || [ -z "$(command -v "${1}")" ] || { [ -f "${1}" ] && ! [ -x "${1}" ]; }; then
  set -- node "$@"
fi

exec "$@"
```

This is the content, so maybe just providing `server.js` would be enough then, to make it more apparent that there is also an entrypoint defined and it is in-use.